pyeafe/__init__.py

```diff
@@ -1 +1,4 @@
 __all__ = ["assembly", "run_tests", "tests"]
+
+from assembly import eafe_assemble
+from run_tests import run_tests
```

Comment:
If I copy the `example1.py` outside of the pyeafe folder, i cannot run it because pyeafe cannot be imported.
```
Traceback (most recent call last):
  File "example1.py", line 7, in <module>
    import pyeafe
  File "/Users/maximilianmetti/Library/Python/2.7/lib/python/site-packages/pyeafe-0.1-py2.7.egg/pyeafe/__init__.py", line 4, in <module>
    from run_tests import run_tests
  File "/Users/maximilianmetti/Library/Python/2.7/lib/python/site-packages/pyeafe-0.1-py2.7.egg/pyeafe/run_tests.py", line 8, in <module>
    from tests import double_sin_3d_test, \
ImportError: No module named tests
Abort trap: 6
```

I fixed this by not including `run_tests` or `tests` in this file at all (neither in `__all__` nor the `from` import).  Dropping the tests makes this work perfectly when I want to use pyeafe in directories outside of the project folder.